swap_meet/electronics.py

```diff
@@ -1,2 +1,6 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+class Electronics(Item):
+    def __init__(self, condition = 0, age = 0):
+        super().__init__("Electronics", condition, age)
```

Comment:
Tiny style point here and elsewhere: when defining default arguments don't include spaces on either side of the equals sign. We would instead write it like:

```
def __init__(self, condition=0, age=0):
```